src/Automation/Data/AsyncScan.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Axe.Windows.Automation.Data
+{
+    /// <summary>
+    /// Contains information, such as ScanResults, related to a particular invocation of the ScanAsync method.
+    /// This object currently contains ScanResults.
+    /// However, additional fields may be added at a future time to add extra information without breaking existing users of the IAsyncScanner API.
+    /// </summary>
+    public class AsyncScan
+    {
+        /// <summary>
+        /// The ScanResults object associated with this scan
+        /// </summary>
+        public ScanResults ScanResults { get; private set; }
```

Comment:
This can be simplified:
```suggestion
        public ScanResults ScanResults { get; }
```

This makes the setter both private and accessible only in the constructor